src/components/drawer/drawer.js

```diff
@@ -11,24 +12,33 @@ export class DrawerComponent {
   querySelectors() {
     this.drawerRoot = this.mountPoint.querySelector(".mdc-drawer--persistent");
     this.menu = this.mountPoint.querySelector(".toolbar__menu");
-
-    this.addAccountButton = this.mountPoint.querySelector(
-      ".drawer__add-account-dialog-activation"
-    );
     this.addTagButton = this.mountPoint.querySelector(
       ".drawer__add-tag-dialog-activation"
     );
-
+    this.accountsMountPoint = this.mountPoint.querySelector(
+      ".drawer__accounts-mountpoint"
+    );
     this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
       ".drawer__more-button"
     );
   }
+  updateAccountData(transaction) {
+    this.accountsComponent.updateAccountData(transaction);
+  }
+
+  initAccountComponent() {
+    this.accountsComponent = new AccountsComponent(this.accountsMountPoint);
+    this.accountsComponent.mount();
+  }
 
   initMDC() {
     this.drawer = new MDCPersistentDrawer(this.drawerRoot);
   }
 
   initMoreBtns() {
+    this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
```

Comment:
the same. you do `this.moreBtnMountPoints = this.mountPoint.querySelectorAll` twice